src/components/progressCircular/js/progressCircularDirective.js

```diff
@@ -79,7 +79,7 @@ function MdProgressCircularDirective($$rAF, $window, $mdProgressCircular, $mdThe
         var mode = hasValue ? MODE_DETERMINATE : MODE_INDETERMINATE;
         var info = "Auto-adding the missing md-mode='{0}' to the ProgressCircular element";
 
-          // $log.debug( $mdUtil.supplant(info, [mode]) );
+        $log.debug( $mdUtil.supplant(info, [mode]) );
```

Comment:
I accidentally introduced this one in a PR yesterday together with that bump in the animation time for the determinate animation.